torch/csrc/jit/codegen/cuda/scheduler/normalization.cpp

```diff
@@ -1,12 +1,14 @@
 #include <torch/csrc/jit/codegen/cuda/scheduler/reduction.h>
 
+#include <torch/csrc/jit/codegen/cuda/arith.h>
 #include <torch/csrc/jit/codegen/cuda/executor_utils.h>
 #include <torch/csrc/jit/codegen/cuda/instrumentation.h>
 #include <torch/csrc/jit/codegen/cuda/ir_all_nodes.h>
 #include <torch/csrc/jit/codegen/cuda/ir_utils.h>
 #include <torch/csrc/jit/codegen/cuda/scheduler/utils.h>
 #include <torch/csrc/jit/codegen/cuda/transform_replay.h>
 
+#include <torch/csrc/jit/codegen/cuda/arith.h>
```

Comment:
Is this needed? Why added twice?

Reply:
Good catch. Thanks! It wasn't needed until we start translating Welford. It's not needed in this PR and I removed it.